ippserver/parsers.py

```diff
@@ -20,6 +20,8 @@ def from_bytes(cls, _data):
 		raise NotImplementedError()
 	def bytes(self):
 		raise NotImplementedError()
+	def __bytes__(self):
+		return self.bytes()
```

Comment:
I think there's probably an assumption that `__bytes__` is human-readable (this is certainly true for `__str__`). It feels like this is a trap, as `bytes()` is only machine-readable!

Reply:
I think not as bytes are not intended to be human readable. You decorate e.g. your fancy Integer class with it to convert it implicitly into a byte object.

Reply:
sry github uses \_ for marking bold text.